src/glossary.rst

```diff
@@ -1708,12 +1754,12 @@ Glossary
 
 
    workflow completion
-      A workflow is deemed complete if there are no more tasks to run,
-      according to the graph, and there are no :term:`incomplete task
-      <incomplete task>` present.
+      A workflow is complete if there are no more tasks to run according
+      to the graph, and there are no :term:`incomplete <output completion>`
+      tasks present in the :term:`n=0 window <n-window>`.
```

Comment:
Incomplete outputs are not sufficient to diagnose workflow completion.

This definition needs to include final task status.

Reply:
If you look at the full sentence, it is very close to correct without listing all the cases. By "no more tasks to run" I include tasks that are ready but held back by (e.g.) xtriggers; and I assumed that none currently running was obvious. Partially satisfied tasks with no required outputs are perhaps the sticking point.

I'll avoid duplication by referring to the glossary entry for "stalled" instead, and make sure that's correct.